test/ocsp/checkari/main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"crypto"
+	_ "crypto/sha256"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/asn1"
+	"encoding/base64"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"math/big"
+	"net/http"
+	"os"
+
+	"github.com/letsencrypt/boulder/core"
+)
+
+// certID matches the ASN.1 structure of the CertID sequence defined by RFC6960.
+type certID struct {
+	HashAlgorithm  pkix.AlgorithmIdentifier
+	IssuerNameHash []byte
+	IssuerKeyHash  []byte
+	SerialNumber   *big.Int
+}
+
+type AuthorityKeyIdentifier struct {
+	KeyIdentifier []byte `asn1:"optional,tag:0"`
+}
+
```

Comment:
Unused?

```suggestion
```

Reply:
Nice catch. I was using this to parse the AuthorityKeyIdentifier extension and extract its inner value, but it turns out that the x509 lib is doing that already.